docs/7.Scan Examples/Terraform Plan Scanning.md

```diff
@@ -15,19 +15,11 @@ Checkov supports the evaluation of policies on resources declared in `.tf` files
 ```json
 terraform init
 terraform plan --out tfplan.binary
-terraform show -json tfplan.binary > tfplan.json
+terraform show -json tfplan.binary | jq > tfplan.json
```

Comment:
I think we need to add somewhere that jq is a prerequisite

Reply:
Added